operator/internal/certrotation/rotation.go

```diff
@@ -74,7 +74,7 @@ func (r *certificateRotation) NewCertificate(signer *crypto.CA, validity time.Du
 		return nil
 	}
 
-	return signer.MakeServerCertForDuration(sets.NewString(sets.List[string](r.Hostnames)...), validity, addClientAuthUsage, addSubject)
+	return signer.MakeServerCertForDuration(sets.Set[string](r.Hostnames), validity, addClientAuthUsage, addSubject)
```

Comment:
I'm getting `redundant type conversion` on this line:

```suggestion
	return signer.MakeServerCertForDuration(r.Hostnames, validity, addClientAuthUsage, addSubject)
```